dc/s2n-quic-dc/events/acceptor.rs

```diff
@@ -0,0 +1,343 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Emitted when a TCP acceptor is started
+#[event("acceptor:tcp:started")]
+#[subject(endpoint)]
+struct AcceptorTcpStarted<'a> {
+    /// The id of the acceptor worker
+    id: usize,
+
+    /// The local address of the acceptor
+    #[builder(&'a s2n_quic_core::inet::SocketAddress)]
+    local_address: SocketAddress<'a>,
+
+    /// The backlog size
+    backlog: usize,
+}
+
+/// Emitted when a TCP acceptor completes a single iteration of the event loop
+#[event("acceptor:tcp:started")]
```

Comment:
Wrong name? Should we have a duplicate event name check in the tooling?

Reply:
Yeah good call. I'll add a check.

Reply:
Added a check in https://github.com/aws/s2n-quic/pull/2379/commits/43e8cb7bf1fb353ee40a7b113e6f38296bcf8f30#diff-7752ab888c9157ce4d2d1e296c5bca325eaaf4904ff9700d7e833ddce0971083R1